p2p/src/libp2p/behaviour.rs

```diff
@@ -31,6 +33,8 @@ pub struct SedaBehaviour {
     pub gossipsub: Gossipsub,
     // TODO: change discovery mechanism
     pub mdns:      mdns::async_io::Behaviour,
+
+    pub kademlia_behaviour: Kademlia<MemoryStore>,
```

Comment:
Why not simply `kademlia`? It is below SedaBehaviour.

```suggestion
    pub kademlia: Kademlia<MemoryStore>,
```